ops/services/app_functions/report_stream_batched_publisher/infra/_var.tf

```diff
@@ -32,10 +35,12 @@ variable "reporting_exception_queue_name" {
   default = "test-event-publishing-exceptions"
 }
 
-variable "report_stream_api_token" {
-  type = string
+variable "report_stream_batch_minimum" {
+  type    = string
+  default = "1"
 }
 
-variable "simple_report_cb_token" {
-  type = string
+variable "report_stream_batch_maximum" {
+  type    = string
+  default = "500"
```

Comment:
Whoops, it looks like one more config variable slipped through manually - this value is currently set to 300 in the function, not 500. We're probably ok to bump to 500 since we switched to the RS async endpoint, but I did want to call out that this would be a change from our current state.

Reply:
Oh that's right, good catch. Just pushed the change.